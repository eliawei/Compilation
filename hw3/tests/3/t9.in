// illegal casting

void main() {
    set[5..6] myset1;
    set[7..10] myset2 = myset1;
    if ((int)myset1 == (int)myset2)
      print("OK");
    printi((int)myset1);
    printi((int)5 + myset1);
}
