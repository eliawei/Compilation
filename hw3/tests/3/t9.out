---end scope---
line 9: type mismatch
